2013-01-26T13:03:41+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:03:42+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:03:42+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:04:30+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:07:44+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:07:45+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:09:00+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:09:03+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:09:03+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:10:28+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:10:29+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:10:30+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:23+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:24+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:24+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:31+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:37+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:38+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:39+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:11:45+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:12:52+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:12:54+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:13:06+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:13:07+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:13:08+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:13:23+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:19:12+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:19:21+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:19:22+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:19:28+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:19:31+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:19:33+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:19:36+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:20:02+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:21:56+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:21:59+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:22:00+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:23:03+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:23:03+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:23:04+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:23:13+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:23:14+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:23:14+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:23:15+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:26:22+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:26:47+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:27:02+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:27:17+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:27:32+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:28:49+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:28:51+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:30:08+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:30:09+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:30:09+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:30:10+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:30:42+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:31:00+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:31:01+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:31:01+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:31:10+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:31:31+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:31:56+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T13:32:00+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T17:37:51+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:37:51+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:37:51+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:37:51+00:00 ERROR {"message":"assets and liabilities should equal up to 0. The difference is 10000","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T17:37:51+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T17:38:13+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:38:14+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:38:14+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:38:14+00:00 ERROR {"message":"assets and liabilities should equal up to 0. The difference is 10000","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T17:38:14+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T17:38:27+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:38:27+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:38:27+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T17:38:27+00:00 ERROR {"message":"assets and liabilities should equal up to 0. The difference is 10000","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T17:38:28+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T19:06:19+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T19:06:20+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T19:35:08+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T19:38:57+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T20:02:57+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:02:57+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:02:57+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:02:57+00:00 ERROR {"message":"No transaction for reference: test \u00f8\u00e6\u00e5 5104367108db5","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(292): helper\\accounting\\utils\\Transactions->getTransactionByRef('test ?????? 510...')\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#2 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#3 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#4 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#5 [internal function]: rpc\\accounting->createTransaction(Array)\n#6 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#7 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#8 {main}"}
2013-01-26T20:02:57+00:00 ERROR {"message":"No transaction for reference: test \u00f8\u00e6\u00e5 5104367108db5","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(292): helper\\accounting\\utils\\Transactions->getTransactionByRef('test ?????? 510...')\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#2 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#3 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#4 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#5 [internal function]: rpc\\accounting->createTransaction(Array)\n#6 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#7 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#8 {main}"}
2013-01-26T20:02:57+00:00 ERROR {"message":"assets and liabilities should equal up to 0. The difference is 10000","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:02:57+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:05:28+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:05:29+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:05:29+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:07:16+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:07:16+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:07:16+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:07:16+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51043774420c7,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:07:17+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:19:48+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:19:48+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:19:48+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:19:48+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51043a643f867,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:19:49+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:20:36+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:20:36+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:20:36+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:20:36+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51043a942ee61,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:20:37+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:21:45+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:21:45+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:21:45+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:21:45+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:21:45+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51043ad8e3d46,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:21:45+00:00 ERROR {"message":"Posting amount should be more than 0, amount was 0","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Vat.class.php(152): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(569): helper\\accounting\\utils\\Vat->resetVatAccounting(14260)\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(126): api\\accounting::resetVat()\n#4 [internal function]: rpc\\accounting->resetVat()\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:21:53+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T20:22:03+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/invoice\/api.class.php(327): api\\accounting::importTransactions(Array, Array)\n#4 \/var\/www\/apps\/invoice\/rpc.class.php(68): api\\invoice::bookkeep('51043aeb1a5f01a...', 12320, NULL)\n#5 [internal function]: rpc\\invoice->post('51043aeb1a5f01a...', 12320)\n#6 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#7 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#8 {main}"}
2013-01-26T20:22:05+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(464): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:22:05+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(518): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:22:05+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(503): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:22:05+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:22:05+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51043aeb9b904,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(431): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:47:13+00:00 ERROR {"message":"Creating default object from empty value","stack":"#0 \/var\/www\/helpers\/accounting\/controller.class.php(485): core\\errorHandler->errorHandler(2048, 'Creating defaul...', '\/var\/www\/helper...', 485, Array)\n#1 \/var\/www\/apps\/accounting\/strategies\/transactions\/ProductCategory.class.php(50): helper\\accounting->mergeTransactions(Object(model\\finance\\accounting\\DaybookTransaction), Object(model\\finance\\accounting\\DaybookTransaction))\n#2 \/var\/www\/apps\/accounting\/api.class.php(412): app\\accounting\\strategies\\transactions\\ProductCategory->getDaybookTransaction()\n#3 \/var\/www\/apps\/invoice\/api.class.php(327): api\\accounting::importTransactions(Array, Array)\n#4 \/var\/www\/apps\/invoice\/rpc.class.php(68): api\\invoice::bookkeep('510440d11a5f01e...', 12320, NULL)\n#5 [internal function]: rpc\\invoice->post('510440d11a5f01e...', 12320)\n#6 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#7 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#8 {main}"}
2013-01-26T20:47:14+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:47:15+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:47:15+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:47:15+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:47:15+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 510440d155acc,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:49:03+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:49:03+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:49:03+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:49:03+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:49:03+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 5104413590dac,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:54:11+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:54:11+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:54:12+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:54:12+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:54:12+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 5104426783f29,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:54:57+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:54:58+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:54:58+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:54:58+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:54:58+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51044294c9a79,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:57:11+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:11+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:11+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:11+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:57:11+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51044325d873f,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:57:17+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:17+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:17+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:17+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:57:17+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 5104432be43e1,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:57:41+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:41+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:41+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T20:57:41+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T20:57:41+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51044343d0630,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T21:03:10+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:03:11+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:03:11+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:03:11+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T21:03:11+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 5104448d37a1c,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T21:09:58+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:09:59+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:10:27+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:10:28+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:10:42+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:10:45+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:10:48+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:10:54+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:10:59+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:02+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:03+00:00 ERROR {"message":"ReferenceText, Invoice 109,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/invoice\/api.class.php(327): api\\accounting::importTransactions(Array, Array)\n#4 \/var\/www\/apps\/invoice\/controller.class.php(201): api\\invoice::bookkeep('510442671a5f01a...', 0)\n#5 [internal function]: app\\invoice->pay('510442671a5f01a...')\n#6 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#7 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#8 {main}"}
2013-01-26T21:11:03+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:12+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:13+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:14+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:15+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:19+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:20+00:00 ERROR {"message":"ReferenceText, Invoice 108,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/invoice\/api.class.php(327): api\\accounting::importTransactions(Array, Array)\n#4 \/var\/www\/apps\/invoice\/controller.class.php(201): api\\invoice::bookkeep('510441351a5f01a...', 0)\n#5 [internal function]: app\\invoice->pay('510441351a5f01a...')\n#6 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#7 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#8 {main}"}
2013-01-26T21:11:20+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:11:21+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:13:19+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:13:19+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:13:19+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:13:19+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T21:13:19+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 510446edc6b30,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T21:36:51+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:36:52+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:36:54+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:36:57+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:36:58+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:36:59+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:37:03+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T21:37:24+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:37:24+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:37:24+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T21:37:24+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T21:37:25+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 51044c92e785b,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T21:37:27+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:01:58+00:00 ERROR {"message":"You don't have permission to access selected page","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:01:59+00:00 ERROR {"message":"You don't have permission to access selected page","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:02:06+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:02:08+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:02:11+00:00 ERROR {"message":"Page assets does not exist in app \\app\\contacts.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:02:12+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:02:16+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:03:47+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:03:47+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:03:47+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:03:47+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:03:47+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 510460d210d51,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:10:13+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:10:13+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:10:13+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:10:14+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:10:14+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 5104625428e1e,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:10:56+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:10:56+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:10:56+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:10:56+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:10:56+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 5104627ea2bcc,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:11:01+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:11:01+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:11:01+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:11:02+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:11:02+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 5104628419b18,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:12:08+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:12:08+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:12:08+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:12:09+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:12:09+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 510462c707f64,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:12:19+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:20+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:20+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:23+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:24+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:30+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:33+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:34+00:00 ERROR {"message":"Page assets does not exist in app \\app\\invoice.","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:36+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:41+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-26T23:12:58+00:00 ERROR {"message":"Unable to create account: 1234","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(463): helper\\accounting\\utils\\Accounts->createAccount(Object(model\\finance\\accounting\\Account))\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(45): api\\accounting::createAccount(Object(model\\finance\\accounting\\Account))\n#2 [internal function]: rpc\\accounting->createAccount(Array)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:12:58+00:00 ERROR {"message":"account 2100 cannot be deleted, as there is associated postings.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(517): helper\\accounting\\utils\\Accounts->deleteAccount(2100)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(83): api\\accounting::deleteAccount(2100)\n#2 [internal function]: rpc\\accounting->deleteAccount(2100)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:12:59+00:00 ERROR {"message":"Account 1234 doesn't exist.","stack":"#0 \/var\/www\/apps\/accounting\/api.class.php(502): helper\\accounting\\utils\\Accounts->getAccountByCode(1234)\n#1 \/var\/www\/apps\/accounting\/rpc.class.php(71): api\\accounting::getAccount(1234)\n#2 [internal function]: rpc\\accounting->getAccount(1234)\n#3 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#4 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#5 {main}"}
2013-01-26T23:12:59+00:00 ERROR {"message":"The referencetext already exists in the accounting","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-26T23:12:59+00:00 ERROR {"message":"ReferenceText, test \u00f8\u00e6\u00e5 510462f949da7,  already exist in this accounting.","stack":"#0 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(110): helper\\accounting\\utils\\Transactions->validateTransaction(Object(model\\finance\\accounting\\DaybookTransaction))\n#1 \/var\/www\/helpers\/accounting\/utils\/Transactions.class.php(87): helper\\accounting\\utils\\Transactions->nonTransactionalInsert(Object(model\\finance\\accounting\\DaybookTransaction), Object(PDO))\n#2 \/var\/www\/apps\/accounting\/api.class.php(430): helper\\accounting\\utils\\Transactions->commit()\n#3 \/var\/www\/apps\/accounting\/rpc.class.php(94): api\\accounting::importTransactions(Object(model\\finance\\accounting\\DaybookTransaction))\n#4 [internal function]: rpc\\accounting->createTransaction(Array)\n#5 \/var\/www\/core\/reqHandler.class.php(150): call_user_func_array(Array, Array)\n#6 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#7 {main}"}
2013-01-27T09:22:05+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-27T09:22:07+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-27T09:22:09+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
2013-01-27T09:22:11+00:00 ERROR {"message":"App not found","stack":"#0 \/var\/www\/webroot\/index.php(224): core\\reqHandler::init()\n#1 {main}"}
